jupyterlab_bigquery/jupyterlab_bigquery/list_items_handler/create_handler.py

```diff
@@ -17,6 +17,22 @@ def _handle_exception(self, e):
     self.finish({"error": {"message": str(e)}})
 
 
+class Handlers():
+  """Singleton for handlers dictionary"""
+
+  _instance = {}
```

Comment:
You should set this to None, and create another variable dict to store the handlers.

Technically objects are dicts, but we should separate that out into a variable